containers/datalab/Dockerfile.in

```diff
@@ -50,7 +50,7 @@ RUN pip install -U tornado==4.3 \
                    argparse==1.2.1 \
                    mock==1.2.0 \
                    requests==2.9.1 \
-                   oauth2client==1.5.2 \
+                   oauth2client==2.0.2 \
```

Comment:
We can't change this; we're hard-bound to 1.5.2 because of dataflow and tensorflow. Please revert.